react-portal-employees/package.json

```diff
@@ -1,12 +1,15 @@
 {
-  "name": "react-portal-employees",
-  "version": "0.1.0",
+  "name": "my-webpack-project",
```

Comment:
why change name?

Reply:
As far as I remember, it appears when I run a "npx webpack-cli init" to generate webpack config